packages/tasks/src/model-libraries.ts

```diff
@@ -703,7 +703,7 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/NVIDIA/NeMo",
 		snippets: snippets.nemo,
 		filter: true,
-		countDownloads: `path_extension:"nemo" OR path:"model_config.yaml"`,
+		countDownloads: `path_extension:"nemo" OR path:"model_config.yaml" OR path_extension:"safetensors"`,
```

Comment:
```suggestion
		countDownloads: `path_extension:"nemo" OR path:"model_config.yaml" OR path_extension:"json"`,
```

better to do config.json IMO: https://huggingface.co/nvidia/canary-qwen-2.5b/blob/main/config.json

Reply:
Done